libraries/androidutils/src/main/kotlin/io/element/android/libraries/androidutils/browser/ChromeCustomTab.kt

```diff
@@ -53,7 +53,7 @@ fun Activity.openUrlInChromeCustomTab(
                 // Disable download button
                 intent.putExtra("org.chromium.chrome.browser.customtabs.EXTRA_DISABLE_DOWNLOAD_BUTTON", true)
                 // Disable bookmark button
-                intent.putExtra("org.chromium.chrome.browser.customtabs.EXTRA_DISABLE_START_BUTTON", true)
+                intent.putExtra("org.chromium.chrome.browser.customtabs.EXTRA_DISABLE_STAR_BUTTON", true)
```

Comment:
I also spot this typo while looking if we could disable the option to minify the ChromeTab during Oidc login, but it appears that it's not possible.

<img width="358" alt="image" src="https://github.com/user-attachments/assets/f6677209-2107-4e3f-b38d-5856c26ea68e" />

Minimized tab does not make any sense here:

<img width="356" alt="image" src="https://github.com/user-attachments/assets/3af721e9-09e8-407f-b690-d557393e13ef" />